commands/upgrade/cluster/command.go

```diff
@@ -113,6 +113,7 @@ func upgradeClusterValidationOutput(cmd *cobra.Command, cmdLineArgs []string) {
 	err := validateUpgradeClusterPreconditions(args, cmdLineArgs)
 
 	if err != nil {
+		client.HandleErrors(err)
```

Comment:
Perhaps out of scope, but all these commands have to learn this correct order of handling errors. Can we extract that to a single call which then encapsulates this knowledge, so that in the future, if something changes, you can make an edit in a single place, instead of having to go through all commands again?

Reply:
Yeah. It started very modular as there are (very few) commands which don't instantiate the `client` package. Might be an option for further optimization.